README.md

```diff
@@ -0,0 +1,54 @@
+# ![Reactotron Logo](./docs/plugin-docs/images/readme/Reactotron-128.png)
+
+[Join our Community Slack](http://community.infinite.red/)
+
+# Introduction
+
+Reactotron is a powerful debugger for React and React Native applications. It provides an easy-to-use interface for developers to monitor their application's **state, network requests, and performance metrics** and can be used for any size of project, from small personal apps to large-scale enterprise applications. The OG debugger at [Infinite Red](https://infinite.red) that we use on a day-to-day basis to build client apps. Additionally, Reactotron is completely open source and free to use, making it an invaluable tool for developers at all levels of experience.
+
+We recommend that you watch [Darin Wilson's](https://github.com/darinwilson) talk at [Chain React](https://chainreactconf.com/): [Chain React 2018: Debugging and Beyond with Reactotron](https://www.youtube.com/watch?v=UiPo9A9k7xc)!
+
+## Installation
+
+On the [Releases](https://github.com/infinitered/reactotron/releases/tag/v2.17.1) page, you can find the latest version of:
+
+- macOS
+- Linux (32-bit & 64-bit)
+- Windows (32-bit & 64-bit)
+
+### Auto Updates
+
+Once you have the application installed it should download and alert you that a new version is available and that new version should be installed on next launch.
```

Comment:
Does this actually work? I haven't experience this yet. I know that V2 was published on Homebrew, but not the V3 yet since it's in beta

Reply:
I have not tested it - I'll remove this for now till we are sure